sdk/ai/azure-ai-projects/README.md

```diff
@@ -158,11 +255,11 @@ agent = project_client.agents.create_agent(
 
 <!-- END SNIPPET -->
 
-In the following sections, we show you code snips in either `toolset` or combination of `tools` and `tool_resources`.   But you are welcome to use another approach.
+In the following sections, we show you sample code in either `toolset` or combination of `tools` and `tool_resources`.   But you are welcome to use another approach.
```

Comment:
I would consider removing "But you are welcome to use another approach."

Reply:
Done!